src/main/java/mate/academy/internetshop/controllers/GetAllOrdersController.java

```diff
@@ -9,16 +9,21 @@
 import mate.academy.internetshop.lib.Injector;
 import mate.academy.internetshop.model.Order;
 import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
 
 public class GetAllOrdersController extends HttpServlet {
```

Comment:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```
ми ж тут витягуємо не абсолютно всі замовлення, що є в базі